### PR TITLE
add baseline support for OpenIndexMaps Specification 1.0.0

### PR DESCRIPTION
**Pull request by srappel**
adds an OpenIndexMaps example that uses the newest version of the [OpenIndexMaps specification (v1.0.0)](https://openindexmaps.org/specification/1.0.0)

### CODE REVIEW

**Comment by srappel** [comment]
Here's the new record in UWM's application: https://geodiscovery.uwm.edu/catalog/ark:-77981-gmgscj87k49

Here's the Stanford example from the fixture, working as expected: https://geodiscovery.uwm.edu/catalog/stanford-fb897vt9938

**Comment by ewlarson** [comment]
I'm seeing a Javascript error when clicking on features/tiles of this OpenIndexMap fixture:

Screenshot 2024-02-01 at 1 39 14 PM

The error appears to be due to GBL's Javascript expecting a `manifestResponse.thumbnail['@id']` value here:
https://github.com/geoblacklight/geoblacklight/blob/main/app/assets/javascripts/geoblacklight/modules/util.js#L29-L30

Stanford's OIM fixture's manifest has this attribute:
https://purl.stanford.edu/ww531pg8163/iiif/manifest

Screenshot 2024-02-01 at 2 43 43 PM

AGSL's OIM fixture's manifest does not:
https://collections.lib.uwm.edu/iiif/2/agdm:4878/manifest.json


----

One work around would be a try/catch block around this section of the code. It's not ideal, but it does allow the AGSL OIM tile images to render, when our handlebars template is modified to check for thumbURL.

Likely, we want much better OIM v1.0 spec support than our current JS feature provides.

**Comment by ewlarson** [comment]
After the potential bug fix is applied:

Screenshot 2024-02-01 at 2 53 23 PM

**Comment by srappel** [comment]
Also see the comments in #1327

**Comment by srappel** [comment]
Making a note here because it's Friday and I don't want to forget: Instead of showing a thumbnail when you click on a sheet, what if it opened a IIIF viewer instead?

File: app/assets/javascripts/geoblacklight/templates/index_map_info.hbs [hunk 1]
```diff
@@ -14,6 +14,15 @@
         <img src="{{thumbnailUrl}}">
       {{/if}}
     {{/if}}
+    {{#if thumbUrl}}
+      {{#if websiteUrl}}
+        <a href="{{websiteUrl}}">
```
Comment (srappel) [inline app/assets/javascripts/geoblacklight/templates/index_map_info.hbs:19]: I was wondering why the thumbnail was not hyperlinked with this new fixture. In the OIM 1.0.0 spec, `digHold` and `websiteUrl` are similar with ambiguous differences, and we've been using digHold primarily. For this fixture, I might include both?

File: app/assets/javascripts/geoblacklight/templates/index_map_info.hbs [hunk 2]
```diff
@@ -23,10 +32,18 @@
         <dt class="col-sm-3">Download:</dt>
         <dd class="col-sm-9"><a href="{{downloadUrl}}">{{downloadUrl}}</a></dd>
       {{/if}}
+      {{#if download}}
+        <dt class="col-sm-3">Download:</dt>
+        <dd class="col-sm-9"><a href="{{download}}">{{download}}</a></dd>
+      {{/if}}
```
Comment (srappel) [inline app/assets/javascripts/geoblacklight/templates/index_map_info.hbs:38]:
```suggestion
      {{/if}}
      {{#if iiifUrl}}
        <dt class="col-sm-3">iiif URL:</dt>
        <dd class="col-sm-9"><a href="{{iiifUrl}}">{{iiifUrl}}</a></dd>
      {{/if}}
```

File: app/assets/javascripts/geoblacklight/templates/index_map_info.hbs (hunk 1 above)
Comment (srappel) [reply app/assets/javascripts/geoblacklight/templates/index_map_info.hbs:19]: I've included both digHold and websiteURL in the OpenIndexMap.

**Comment by srappel** [comment]
Wrapping up for winter sprint 2024:

- [ ] Create an IndexMaps directory in fixtures
- [ ] Include the existing fixtures in that directory
- [ ] Fix the linkage between the solr documents and the OIM GeoJSONs
- [ ] Update the README file in the solr documents
- [ ] Someone other than @srappel review

**Comment by srappel** [comment]
Note that [4a9438a](https://github.com/geoblacklight/geoblacklight/pull/1333/commits/4a9438a12679bec947aa4e0a6ef9a79eaa077257) will probably break things in testing because solr documents will point to github raw geojson files on the main branch of geoblacklight that won't actually exist until this is merged.

**Comment by srappel** [comment]
Yeah, it's breaking CI:

> Failed examples:
> 
> rspec ./spec/features/index_map_spec.rb:9 # Index map displays index map viewer (polygon)
> rspec ./spec/features/index_map_spec.rb:21 # Index map displays index map viewer (points)

**Comment by karenmajewicz** [comment]
Can we add the OIM fixtures in a separate PR?

**Comment by srappel** [comment]
ok @ewlarson @kgjenkins this is passing CI and ready for review!

File: app/assets/javascripts/geoblacklight/templates/index_map_info.hbs [hunk 3]
```diff
@@ -14,6 +14,15 @@
         <img src="{{thumbnailUrl}}">
       {{/if}}
     {{/if}}
+    {{#if thumbUrl}}
+      {{#if websiteUrl}}
+        <a href="{{websiteUrl}}">
+          <img src="{{thumbUrl}}" class="img-responsive">
```
Comment (ewlarson) [inline app/assets/javascripts/geoblacklight/templates/index_map_info.hbs:20]: Let's change the `img-responsive` CSS class names in this whole file to `img-fluid`.

Bootstrap v4 migrated to `img-fluid` from `img-responsive` and we just never fixed that in GBL: https://getbootstrap.com/docs/4.6/migration/#images

- [ ] thumbUrl class
- [ ] thumbnailUrl (used just above here)

**Review by ewlarson — CHANGES_REQUESTED** [review]
🥇 / Looks really good to me, except for one teeny tiny small change request.

File: app/assets/javascripts/geoblacklight/templates/index_map_info.hbs (hunk 3 above)
Comment (srappel) [reply app/assets/javascripts/geoblacklight/templates/index_map_info.hbs]: done!

**Review by ewlarson — APPROVED** [review]

**Comment by ewlarson** [comment]
👏 / Nice work here!

**Pull request merged**